201-machine-learning-advanced/azuredeploy.json

```diff
@@ -25,6 +25,24 @@
         "description": "Specifies the sku, also referred as 'edition' of the Azure Machine Learning workspace."
       }
     },
+    "identityType": {
+      "type": "string",
+      "defaultValue": "systemAssigned",
+      "allowedValues": [
+        "systemAssigned",
+        "userAssigned"
+      ],
+      "metadata": {
+        "description": "Specifies the identity type of the Azure Machine Learning workspace."
+      }
+    },
+    "primaryUserAssignedIdentity": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Specifies the user assigned identity that represents the Azure Machine Learing workspace."
```

Comment:
is this the name or the resourceId?  If the latter use separate parameters for sub,rg,name

Reply:
separate the parameters for rg and name